pages/contact.vue

```diff
@@ -138,6 +138,33 @@ export default {
       description:
         'Create custom online forms with ease using Formester - the top-rated online form builder. Contact us today to start building your perfect form!',
       url: 'https://formester.com/contact/',
+      operatingSystem: 'all',
+      browserRequirements: 'Requires Javascript and HTML5 support',
```

Comment:
does it really?

Reply:
@ankitsinghaniyaz, "browerRequirements" is a custom property that we are defining but it does not serve any official standard Schema purpose. This info could be useful for users or other developers who need to know the technical requirements for accessing and using the web page effectively. 

If you suggest we can remove this.